openforcefield/topology/molecule.py

```diff
@@ -4697,6 +4699,132 @@ def add_conformer(self, coordinates):
+                    'installed. Try conda install -c conda-forge nglview.'
+                )
+            if self.conformers:
+                trajectory_like = _OFFTrajectoryNGLView(self)
+                widget = nv.NGLWidget(trajectory_like)
+                return widget
+            else:
+                raise ValueError(
+                    'Visualizing with NGLview requires that the molecule has '
+                    'conformers.'
+                )
+        if backend == 'rdkit':
+            if RDKIT_AVAILABLE:
+                from rdkit.Chem.Draw import IPythonConsole
+                return self.to_rdkit()
+            else:
+                backend = 'openeye'
+        if backend == 'openeye':
+            if OPENEYE_AVAILABLE:
+                from openeye import oedepict
+                from IPython.display import Image
+
+                oemol = self.to_openeye()
+
+                opts = oedepict.OE2DMolDisplayOptions(width, height, oedepict.OEScale_AutoScale)
+
+                oedepict.OEPrepareDepiction(oemol)
+                img = oedepict.OEImage(width, height)
+                display = oedepict.OE2DMolDisplay(oemol, opts)
+                oedepict.OERenderMolecule(img, display)
+                png = oedepict.OEWriteImageToString("png", img)
+                return Image(png)
+
+        raise ValueError('Could not find an appropriate backend')
+
+    def _ipython_display_(self):
+        from IPython.display import display
+        try:
+            return display(self.visualize(backend='nglview'))
+        except (ImportError, ValueError):
+            pass
+
+        try:
+            return display(self.visualize(backend='rdkit'))
+        except ValueError:
+            pass
+
+        try:
+            return display(self.visualize(backend='openeye'))
+        except ValueError:
+            pass
+
+
+try:
+    from nglview import Trajectory as _NGLViewTrajectory
+except ImportError:
+    _NGLViewTrajectory = object
+
+
+class _OFFTrajectoryNGLView(_NGLViewTrajectory):
+    """
+    Handling conformers of an OpenFF Molecule as frames in a trajectory. Only
+    to be used for NGLview visualization.
+
+    Parameters
+    ----------
+    molecule : openforcefield.topology.Molecule
+        The molecule (with conformers) to visualize
+    """
+    def __init__(self, molecule):
+        self.molecule = molecule
+        self.ext = "pdb"
+        self.params = {}
+        self.id = str(uuid.uuid4())
+
+    def get_coordinates(self, index):
+        return self.molecule.conformers[index] / unit.angstrom
+
+    @property
+    def n_frames(self):
+        return len(self.molecule.conformers)
+
+    def get_structure_string(self):
+        memfile = StringIO()
+        self.molecule.to_file(memfile, "pdb")
+        memfile.seek(0)
+        block = memfile.getvalue()
+        # FIXME: Prevent multi-model PDB export with a keyword in molecule.to_file()?
```

Comment:
I think this is fine for this PR. @mattwthompson can you open up a new issue for adding this functionality in the PDB exporters?

Reply:
I believe this was fixed in https://github.com/openforcefield/openforcefield/pull/579 but we didn't update this comment. Somebody back me up on this actually being fixed before I merge?

Reply:
I'd say 579 solved a different issue (RDKit not being able to generate multi-model PDBs). What I noticed there was an API-level way to tell the toolkit we don't want all conformers written to the PDB, just the first one. Something like this:

```python
self.molecule.to_file(memfile, "pdb", first_model_only=True)
```

or 
```python
self.molecule.to_file(memfile, "pdb", conformers=[0])
# conformers can be None/'all', or a list of indices or slice?
```

Reply:
Moved this to #585